otter_buddy/cogs/interview_match.py

```diff
@@ -121,15 +123,34 @@ async def send_pair_message(self, otter_one: discord.User, otter_two: discord.Us
             conn = EmailConn()
             conn.send_mail(email, "Interview buddy", message)
     
-    def make_pairs(self, week_otter_pool: list, placeholder: discord.Member):
+    def make_pairs(self, week_otter_pool: list, placeholder: discord.Member) -> List[Tuple[discord.Member, discord.Member]]:
         week_otter_pairs: list = []
+        week_otter_collaborator: list = []
+        week_otter_member: list = []
         if len(week_otter_pool) % 2 == 1:
             week_otter_pool.append(placeholder)
+
+        for otter in week_otter_pool:
+            str_otter_roles = [str(otter_role) for otter_role in otter.roles]
```

Comment:
Putting the type of the variable as prefix it's a bad practice, if you are using types already why not declaring this variable as a array of strings?

Reply:
Any updates on this?